src/main/resources/graphics/scenery/backends/vulkan/shaders/DeferredLighting.frag

```diff
@@ -231,27 +305,27 @@ void main()
 	} else {
 		vec2 newTexCoord;
 		// color
-		if(textureCoord.x < 0.25 && textureCoord.y < 0.5 ) {
-			FragColor = Albedo;
-		}
-		// specular
-		if(textureCoord.x > 0.25 && textureCoord.x < 0.5 && textureCoord.y < 0.5) {
-		    FragColor = vec4(Specular, Specular, Specular, 1.0);
-		}
-		// depth
-		if(textureCoord.x > 0.5 && textureCoord.y < 0.5) {
-		    float near = 0.5f;
-		    float far = 1000.0f;
-		    vec3 linearizedDepth = vec3((2.0f * near) / (far + near - Depth * (far - near)));
-			FragColor = vec4(linearizedDepth, 1.0f);
-		}
-		// normal
-		if(textureCoord.x > 0.5 && textureCoord.y > 0.5) {
-			FragColor = vec4(Normal, 1.0f);
-		}
-		// position
-		if(textureCoord.x < 0.5 && textureCoord.y > 0.5) {
-			FragColor = vec4(FragPos, 1.0f);
-		}
+        if(textureCoord.x < 0.25 && textureCoord.y < 0.5 ) {
```

Comment:
It looks like the indentation here is not as it should be, could you check that please? Thanks!